lib/chains/elementChain.js

```diff
@@ -89,7 +93,7 @@ const toString = (data, nameOnly = false) => {
 				break;
 			default: {
 				const matches = data.comparator.props.map(
-					one => `  ${getAllSymbols[one.name]} ${getAllSymbols[one.type]} ${getAllSymbols[one.val] || one.val}`
+					one => `  ${one.name} ${one.type} ${VALUE_M[one.val] || one.val}`
```

Comment:
This does not look correct (unless overwritten in later commits). The other line should have been taken.

Reply:
there are no symbols anymore, only VALUE

Reply:
right.
@AntonSBilan what about validationKeys.js? 

Not sure if some of those could appear in some of the output logs. If yes they will need to be translated.